src/graphql/main/schema.graphql

```diff
@@ -91,6 +91,12 @@ input CaptchaRequestAuthCodeInput {
 """
 scalar CentAmount
 
+type CentAmountPayload {
+  amount: CentAmount
+  currencyUnit: ExchangeCurrencyUnit
```

Comment:
This was originally to support the dual return (usd/btc) from the `LnInvoiceFeeProbe` mutation but I've since added back the WalletCurrency check to that one... which make me realize now that we're missing a `LnUsdInvoiceFeeProbe` mutation

Reply:
Fixed: https://github.com/GaloyMoney/galoy/pull/1237/commits/f21087523b8e96cebc65b5c63ce448347db7abed & https://github.com/GaloyMoney/galoy/pull/1237/commits/44ca34e462e10d55af6e54152c2bdc91ef293a36